src/mail/mjml/partial/installment_table.mjml

```diff
@@ -0,0 +1,59 @@
+<mj-section mj-class="bg--blue-100" padding="0 50px 0 50px">
+  <mj-column>
+    <mj-text padding="0" font-size="16px">
+      <strong>{% trans "Payment schedule" %}</strong>
+    </mj-text>
+  </mj-column>
+</mj-section>
+<mj-section mj-class="bg--blue-100">
+  <mj-column>
+    <mj-text padding="0 50px 0 50px">
+      <div style="background-color:#ffffff; border: 1px solid #B8C7E3; border-radius: 8px; overflow: hidden;">
+        <table style="font-size: 14px; width: 100%; border-collapse: collapse; overflow: hidden;">
+          {% for installment in order_payment_schedule %}
+          {% with amount=installment.amount|format_currency_with_symbol installment_date=installment.due_date|date:"SHORT_DATE_FORMAT" %}
+          <tr style="{% if not forloop.last %}border-bottom: 1px solid #B8C7E3;{% endif %} {% if targeted_installment_index == forloop.counter0 %}background-color: #EDF5FA; {% endif %}">
+            <td style="padding: 10px 10px 10px 20px;">
+              <mj-text>{{ forloop.counter }}</mj-text>
+            </td>
+            <td style="padding:10px;">
+              <mj-text><strong>{{ amount }}</strong></mj-text>
+            </td>
+            <td style="padding:10px;">
+              <p>{%trans "Withdrawn on" %} {{ installment_date }}</p>
```

Comment:
I think you should use blocktranslate instead of `trans` then add `installment_date` as variable. In some languages, the installment_date may be display before text.